Sources/protoc-gen-swift/Context.swift

```diff
@@ -188,6 +197,21 @@ class Context {
   init(request: CodeGeneratorRequest) throws {
     self.request = request
     self.options = try GeneratorOptions(parameter: request.parameter)
+
+    if request.hasCompilerVersion {
+      let compilerVersion = request.compilerVersion;
+      // Expect 3.1.x or 3.2.x - Yes we have to rev this with new release, but
+      // that seems like the best thing at the moment.
+      let isExpectedVersion = (compilerVersion.major == 3) &&
+        (compilerVersion.minor >= 1) &&
+        (compilerVersion.minor <= 2)
```

Comment:
Maybe?  Version.swift is a little odd to me in that it has version info as well as help which directly maps to the command line support in main.  It might make sense to just have version numbers there and move all the other stuff into main since it isn't _version info_?